root-config/package.json

```diff
@@ -1,31 +1,34 @@
     "start:react-apps:main-featured-product-app": "cd ../react-apps/main-featured-product-app/dist && npx http-server . --cors -p 9003",
+    "start:angular-apps:main-categories-app": "cd ../angular-apps/main-categories-app/dist/main-categories-app && npx http-server . --cors -p 9002",
     "lint": "eslint src --ext js,ts,tsx",
     "test": "cross-env BABEL_ENV=test jest --passWithNoTests",
     "format": "prettier --write .",
     "check-format": "prettier --check .",
-    "install-all": "npm run install:root-config && npm run install:angular-apps:contact-form-app && npm run install:angular-apps:common-header-app && npm run install:react-apps:common-hero-search-app && npm run install:react-apps:main-featured-product-app && npm run install:react-apps:common-footer-app",
+    "install-all": "npm run install:root-config && npm run install:angular-apps:contact-form-app && npm run install:angular-apps:common-header-app && npm run install:react-apps:common-hero-search-app && npm run install:react-apps:main-featured-product-app && npm run install:react-apps:common-footer-app && npm run install:angular-apps:main-categories-app",
     "install:root-config": "npm i",
     "install:angular-apps:contact-form-app": "cd ../angular-apps/contact-form-app/ && npm i",
     "install:angular-apps:common-header-app": "cd ../angular-apps/common-header-app/ && npm i",
     "install:react-apps:common-hero-search-app": "cd ../react-apps/common-hero-search-app/ && npm i",
     "install:react-apps:common-footer-app": "cd ../react-apps/common-footer-app/ && npm i",
     "install:react-apps:main-featured-product-app": "cd ../react-apps/main-featured-product-app/ && npm i",
-    "build": "npm run build:angular-apps:contact-form-app && npm run build:angular-apps:common-header-app && npm run build:react-apps:common-hero-search-app && npm run build:react-apps:common-footer-app && npm run build:react-apps:main-featured-product-app",
+    "install:angular-apps:main-categories-app": "cd ../angular-apps/main-categories-app/ && npm i",
+    "build": "npm run build:angular-apps:contact-form-app && npm run build:angular-apps:common-header-app && npm run build:react-apps:common-hero-search-app && npm run build:react-apps:common-footer-app && npm run build:react-apps:main-featured-product-app && npm run build:angular-apps:main-categories-app",
     "build:angular-apps:contact-form-app": "cd ../angular-apps/contact-form-app/ && npm run build:single-spa:contact-form-app",
     "build:angular-apps:common-header-app": "cd ../angular-apps/common-header-app/ && npm run build:single-spa:common-header-app",
-    "build:react-apps:common-hero-search-app": "cd ../react-apps/common-hero-search-app/ && webpack --config ./webpack.config.js",
+    "build:react-apps:common-hero-search-app": "cd ../react-apps/common-hero-search-app/ && webpack --config ./webpack.config.js --env API_ENDPOINT=http://localhost:5000",
```

Comment:
Please merge code from develop branch and you will see the env already added to the script.